src/braket/circuits/gate.py

```diff
@@ -198,7 +198,7 @@ def _to_openqasm(
 
         return (
             f"{inv_prefix}{power_prefix}{control_prefix}"
-            f"{self._qasm_name}{param_string} {', '.join(qubits)};"
+            f"{self._qasm_name}{param_string}{','.join([f' {qubit}' for qubit in qubits])};"
```

Comment:
We should be able to just do `qubits` here instead of `map(str, qubits)`, since `qubits` already has the serialization properties-based formatting applied to the int values

Reply:
Both of the suggestions will leave no space between `param_string` and the qubits, no? Being explicit is nice, but I think JC's implementation is elegant enough.